pyswagger/contrib/client/requests.py

```diff
@@ -43,12 +43,12 @@ def request(self, req_and_resp, opt={}):
             files=file_obj
         )
         rq = self.__s.prepare_request(rq)
-        rs = self.__s.send(rq)
+        rs = self.__s.send(rq, stream=True)
 
         resp.apply_with(
             status=rs.status_code,
             header=rs.headers,
-            raw=rs.text
+            raw=six.StringIO(rs.content).getvalue()
```

Comment:
use StringIO instead of text in case the response is binary format, e.g. protobuf